atlp-rwanda/eagles-ec-be
# #187419184 A buyer should be able to wish a product

#### What does this PR do?
This PR implements wishlist feature for buyers

#### Description of Task to be completed?
- a buyer can add products to their wishes
- a buyer can see all products he/she wished
- a buyer can remove a product from their wishes
- sellers can see all wishes made on their own products

#### How should this be manually tested?
1. Clone the repository.
2. Checkout to the branch `ft-wishlist-#187419184`.
3. Run `npm install` to install dependencies.
4. Run `npm run migrate` and `npm run seed`
5. Run `npm run dev` to start the development server.
6. Open a web browser and navigate to the `/docs` endpoint (e.g., `http://localhost:PORT/docs`).
7. Verify that the Swagger UI loads correctly.
8. First authorize by logging in any user and use the token provided (a buyer and a seller for product wishes)
9. Explore the endpoints for wishes and verify that everything is working.

#### What are the relevant pivotal tracker stories?

[Delivers #187419184]
